### ISSUE TITLE
[MU4 Issue] [Accessibility] Screen reader doesn't read lyrics, slurs and lines in score

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9589]
https://user-images.githubusercontent.com/90187801/139092383-a5d84223-e448-4d95-b2cc-722e0bad5243.mp4

**Desktop (please complete the following information):**
 - OS: Win10/Mac11/LinuxMint20
